src/style.css

```
@@ -1 +1,50 @@
 /* add styles here */
+body {
+  display: flex;
+  margin: 0;
+  background-color: gray;
+}
+
+.display {
+  display: flex;
+  margin: 0 auto;
+  padding-bottom: 6px;
+  align-items: center;
+  background-color: rgba(255, 255, 255, 1);
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.stars__star {
+  width: 12px;
+  height: 12px;
+  background-image: url(images/star.svg);
+  background-size: cover;
+  margin: 2px;
+  margin-right: 6px;
+}
+
+.stars--1 :nth-child(1) {
```

```suggestion
.stars--1 :nth-child(1),
.stars--2 :nth-child(-n + 2),
... {
```
Consider using commas to group selectors